### PR TITLE
Seperates the surplus equipment vendor into a clothing/armor vendor

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
semi temporary measure until someone does tabs, adds a clothing vendor to the game for more organized stuff and so it doesn't have literally 50 or so items.

## Why It's Good For The Game
so the surplus equipment vendor isn't horrifically bloated

## Changelog
:cl:
add: Added new clothing vendor
tweak:added clothing vendor and separated the surplus equipment vendor into a clothing/armor vendor.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -60. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -637,8 +637,64 @@
 
 
 /obj/machinery/vending/uniform_supply
-	name = "\improper Surplus Equipment Vendor"
-	desc = "A automated equipment rack hooked up to a colossal storage of uniforms and accessories."
+	name = "\improper Surplus Armor Equipment Vendor"
+	desc = "A automated equipment rack hooked up to a colossal storage of armor and accessories."
+	icon_state = "marineuniform"
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:642]: armor_supply if its an armor vendor

Comment (Pariah919) [reply code/game/objects/machinery/vending/marine_vending.dm]: done

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 2]
```diff
@@ -636,9 +636,66 @@
...
 
 
+/obj/machinery/vending/armor_supply
+	name = "\improper Surplus Armor Equipment Vendor"
+	desc = "A automated equipment rack hooked up to a colossal storage of armor and accessories."
+	icon_state = "marineuniform"
+	icon_vend = "marineuniform_vend"
+	icon_deny = "marineuniform"
+	var/squad_tag = ""
+
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/marine_vending.dm:646]: why

File: _maps/map_files/Theseus/TGS_Theseus.dmm [hunk 3]
```diff
@@ -38091,6 +38091,12 @@
 	dir = 1
 	},
 /area/mainship/living/briefing)
+"vqs" = (
+/obj/machinery/vending/armor_supply,
+/turf/open/floor/mainship{
+	icon_state = "green"
+	},
```
Comment (TiviPlus) [inline _maps/map_files/Theseus/TGS_Theseus.dmm:38098]: bad varedit

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 4]
```diff
@@ -708,31 +759,6 @@
 		/obj/item/clothing/shoes/white = 50,
 		/obj/item/clothing/gloves/white = 50,
 		/obj/item/clothing/suit/modular = 20,
```
Comment (Hughgent) [inline code/game/objects/machinery/vending/marine_vending.dm:761]: This is now in both vendors. please fix.

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 5]
```diff
@@ -650,14 +702,12 @@
 		/obj/item/clothing/shoes/marine = 20,
 		/obj/item/clothing/under/marine/standard = 20,
 		/obj/item/clothing/under/marine/jaeger = 30,
+		/obj/item/clothing/suit/storage/marine/harness = 50,
```
Comment (Hughgent) [inline code/game/objects/machinery/vending/marine_vending.dm:705]: The harness, while yes not actually providing any armor whatsoever, fits into the 'armor' slot so to speak.

also, don't make it 50 of them. Keep it at 5.

**Review by Hughgent — CHANGES_REQUESTED** [review]

**Review by Hughgent — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]
This can be straight merged it doesnt need tmc

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-69). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**